src/com_tjnotifications/admin/models/notification.php

```diff
@@ -595,4 +595,61 @@ public function deleteBackendConfigs($backendConfigIdsToBeDeleted)
 			}
 		}
 	}
+
+	/**
+	 * Method to replace tags if they are changed
+	 *
+	 * @param   array   $template  This is single template array of data
+	 * @param   string  $client    client like com_jgive
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function updateTemplates($template, $client)
+	{	
+		Table::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_tjnotifications/tables');
```

Comment:
can this be simpler as createTemplates?

Reply:
This function will be used only one time hence I have not divided it.

Also, Instead of writing new function I had tried  to use exsiting createTemplates() function, but while using this function we will need to write code for taking temaplate id in every extension, which is repeatable work.

Also, in update case createTemplates() function not returning templatId and hence it is not working.